compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -113,7 +284,6 @@ void wquant_with_minmax_per_layer(CircleConst *node, float min, float max, float
 
 void wdequant_with_minmax_per_layer(CircleConst *node, float scaling_factor, float nudged_min)
 {
-
```

Comment:
I think this change was from previous commit with addition line, but added cause of my comment.
As this is not related with this PR, can you please separate this change to another PR ?

Reply:
thank you for your advice. I will apply it.

Reply:
Maybe you forgot to remove this change?